pytroll_collectors/s3stalker.py

```diff
@@ -93,17 +189,33 @@ def set_last_fetch(timestamp):
     DatetimeHolder.last_fetch = timestamp
 
 
+def get_last_fetch():
+    """Get the last fetch time."""
+    return DatetimeHolder.last_fetch
+
+
+def create_messages_for_recent_files(bucket, config):
+    """Create messages for recent files and return."""
+    logger.debug("Create messages for recent files...")
+    time_back = config['timedelta']
+    logger.debug("time_back = %s", str(time_back))
+    subject = config['subject']
+    pattern = config.get('file_pattern')
+    with sleeper(2.5):
+        set_last_fetch(datetime.now(tz.UTC) - timedelta(**time_back))
+        s3_kwargs = config['s3_kwargs']
+        logger.debug('s3_kwargs: %s', str(s3_kwargs))
```

Comment:
This will print also the credentials, right? Not good if so...

Reply:
Yes, think you are right, will remove